Formula/pari.rb

```diff
@@ -1,8 +1,8 @@
 class Pari < Formula
   desc "Computer algebra system designed for fast computations in number theory"
   homepage "https://pari.math.u-bordeaux.fr/"
-  url "https://pari.math.u-bordeaux.fr/pub/pari/unix/pari-2.11.4.tar.gz"
-  sha256 "bfc88fc4f7352f4840e6e352c72f0369cbea8a45403b1834a6269f3709970b1c"
+  url "https://pari.math.u-bordeaux.fr/pub/pari/unix/pari-2.13.0.tar.gz"
+  sha256 "c811946de9d2c1ed0e97ff08e80d966f9a0b55848b7688406fab229e3948ba93"
   license "GPL-2.0"
```

Comment:
```suggestion
  license "GPL-2.0-or-later"
```

```
   5 PARI/GP is free software; you can redistribute it and/or modify it under the
   6 terms of the GNU General Public License as published by the Free Software
   7 Foundation; either version 2 of the License, or (at your option) any later
   8 version. It is distributed in the hope that it will be useful, but WITHOUT
   9 ANY WARRANTY WHATSOEVER.
```

https://pari.math.u-bordeaux.fr/cgi-bin/gitweb.cgi?p=pari.git;a=blob;f=src/headers/paridecl.h;h=999091e58da12265599d68176d2cabe213fe5d48;hb=2ff9ce95290f1fb3b03dae4e215583613a0c70c4